app/controllers/concerns/owner_exporter.rb

```diff
@@ -86,7 +86,39 @@ def owner_mailing_list_csv(owner)
     csv_string
   end
 
+  def admin_searches_csv(start_date, end_date)
```

Comment:
Can we move this to a different concern file for admin-initiated exports?  Something like `AdminExporter`?

Reply:
Yep, done 👍